src/Cli/dotnet/Commands/Run/VirtualProjectBuildingCommand.cs

```diff
@@ -891,6 +891,7 @@ public static void WriteProjectFile(
                 writer.WriteLine("""
                         <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
                         <DisableDefaultItemsInProjectFolder>true</DisableDefaultItemsInProjectFolder>
+                        <RestoreUseStaticGraphEvaluation>false</RestoreUseStaticGraphEvaluation>
```

Comment:
Should we try looking through the property directives and seeing if any of them are an explicit enable of static graph, and give a more reasonable error if so? Or is it too niche?